src/stylesheets/themes/hugo-terminal-orange/variables.scss

```diff
@@ -0,0 +1,25 @@
+$orange: rgb(255,168,106);
+$orange-200: #ffd1ac;
+$gray-000: #181818;
+$gray-100: #2B303B;/*comment header bg*/
```

Comment:
There should be a space between the comment character and the comment content, for example: 

`/* comment header bg */`

> This is not required, but I recommend it.